strangerstudios/paid-memberships-pro
# Remove slash in all uses of admin_url() function #2172

### All Submissions:






### Changes proposed in this Pull Request:
- Remove all the slash leading / in uses of the [admin_url function](https://developer.wordpress.org/reference/functions/admin_url/).
- Test the functionality


Resolves XXX.

### How to test the changes in this Pull Request:

1. There are few places I have made changes but to check one of them Go to Memberships>Settings>Discount Code>Add new discount code> then click on the Cancel button
2. I have tested on my local development wordpress environment and everything looks good

### Other information:




### Changelog entry

> I have removed all slash in all uses of admin_url() function #2172 in the mentioned files
